kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/sidePanels/LessonResourceSelection/subPages/PreviewSelectedResources/PreviewSelectedExercise.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+
+  <div>
+    <QuestionsAccordion
+      :questions="questions"
+      :selectedQuestions="[]"
+      :getQuestionContent="() => activeResourceMap"
+      :isCheckBoxVisible="false"
+    />
+  </div>
+
+</template>
+
+
+<script>
+
+  import QuestionsAccordion from '../../../../../../common/QuestionsAccordion.vue';
+  import useFetchExercise from '../../../../../../../composables/useFetchExercise';
+
+  export default {
+    name: 'PreviewSelectedExercise',
```

Comment:
I think a better name for this component (and file) would be just `PreviewExcercise` to be consistent with the name of the `PreviewContent` component, and that both of them are inside the `PreviewSelectedResources` folder